share/rocm/cmake/ROCMCreatePackage.cmake

```diff
@@ -181,7 +181,7 @@ macro(rocm_set_cpack_gen)
             string(TOUPPER $ENV{ROCM_PKGTYPE} CPACK_GENERATOR) # PKGTYPE is typically lower case
         else()
             # Otherwise see what we can find
-            message(INFO "rcom_set_cpack_gen didn't find PKGTYPE in environment")
+            message(INFO "rocm_set_cpack_gen didn't find PKGTYPE in environment")
```

Comment:
Should it also say `ROCM_PKGTYPE` rather than `PKGTYPE`?

Reply:
Yes, it should. Updated.